app/models/listing_unit.rb

```diff
@@ -0,0 +1,19 @@
+# == Schema Information
+#
+# Table name: listing_units
+#
+#  id                  :integer          not null, primary key
+#  unit_type           :string(32)       not null
+#  translation_key     :string(64)
+#  transaction_type_id :integer
+#  created_at          :datetime         not null
+#  updated_at          :datetime         not null
+#
+
+class ListingUnit < ActiveRecord::Base
+  attr_accessible(
+    :transaction_type_id,
+    :unit_type,
```

Comment:
Could this be renamed this to 'type'
It would remove some key-renames and if Unit-entity doesn't need it to be named unit_type, why ListingUnit need this longer form?